build.gradle

```diff
@@ -43,19 +44,23 @@ dependencies {
 
   implementation 'org.springframework.session:spring-session-data-redis'
 
+  implementation platform('io.awspring.cloud:spring-cloud-aws-dependencies:3.1.1')
+  implementation 'io.awspring.cloud:spring-cloud-aws-starter-parameter-store'
```

Comment:
Should we use then AwsCredentialsProvider and AwsRegionProvider from Spring Cloud AWS?

Reply:
Probably we should, I'll change the code accordingly.